src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -75,10 +75,38 @@
 				</NcActions>
 			</div>
 
-			<!-- New Conversation -->
+			<!-- Actions -->
+			<div class="actions">
+				<NcActions class="conversations-actions">
+					<template #icon>
+						<DotsVertical :size="20" />
```

Comment:
It's up to @nextcloud/designers , but I'd go here with `DotsHorizontal` or `Plus`

And buttons order should be defined:
* List open conversations
* Create a new conversation
or 
* Create a new conversation
* List open conversations